src/fermilib/utils/_operator_utils.py

```diff
@@ -102,3 +111,102 @@ def commutator(operator_a, operator_b):
             isinstance(operator_b, (QubitOperator, FermionOperator))):
         return operator_a * operator_b - operator_b * operator_a
     raise TypeError('Operator of invalid type.')
+
+
+def save_operator(operator, file_name=None, data_directory=None):
+    """Save FermionOperator or QubitOperator to hdf5 file.
+
+    Args:
+        operator: An instance of FermionOperator or QubitOperator.
+        file_name: The name of the saved hdf5 file. If not provided, the unix
+            time when the saving happens will be used.
+        data_directory: Optional data directory to change from default data
+            directory specified in config file.
+
+    Raises:
+        OperatorUtilsError: Not saved, file already exists.
+        TypeError: Operator of invalid type.
+    """
+    file_path = get_file_path(file_name, data_directory)
+
+    if os.path.isfile(file_path):
+        raise OperatorUtilsError("Not saved, file already exists.")
+
+    if isinstance(operator, FermionOperator):
```

Comment:
Would you mind adding a "not implemented error" specifically when somebody tries to save an InteractionOperator or InteractionRDM? Eventually (but not now) we may want to include that.